tests/dialogue_manager/test_dialogue_policy.py

```diff
@@ -206,9 +210,12 @@ def test_next_action_accept_recommendation(
     ],
 )
 @mock.patch(
-    "moviebot.dialogue_manager.dialogue_policy.set", mock.MagicMock(wraps=list)
+    "moviebot.dialogue_manager.dialogue_policy.rb_dialogue_policy.set",
+    mock.MagicMock(wraps=list),
 )
-def test__generate_examples(policy: DialoguePolicy, results, slot, expected):
+def test__generate_examples(
+    policy: RuleBasedDialoguePolicy, results, slot, expected
+):
     random.seed(42)
     examples = policy._generate_examples(results, slot)
     assert examples == expected
```

Comment:
Would it be possible to have some tests for these two new policies?

Reply:
I created a follow-up issue #229 for that.